### PR TITLE
Add back playground-hero.png image and make frontmatter `version` and `keywords` consistent

### PR DESCRIPTION
**Pull request by jimmychu0807**
- Add back `playground-hero.png`. It has been missing this image. This solves a warning.
- Frontmatter `version` are all converted to string, `3.0` is quoted to be `'3.0'`. This solves a warning.
- Frontmatter `keywords` are all covered to array of string. This solves a warning.

### CODE REVIEW

**Comment by imadarai** [comment]
Thanks @jimmychu0807 ! 
Adding @rmnprkrl to review as it touches a lot of files and some infra-changes that he's been working on.

**Comment by jimmychu0807** [comment]
@imadarai 

1. On playground image, my guess is that we categorized images into sub-folders at one point, but `playground-hero.png` was unintentionally deleted. If `playground-hero.png` is to be deleted intentionally, it should be replaced with another hero image to be loaded at that position. Right now, that spot is blank.

   Screenshot 2021-12-14 at 12 29 34 
   https://docs.substrate.io/

2. Frontmatter data type should only be under one type. For `version`, it used to have both `3` as number type and `'3.0'` as string type. They are now all updated to strings. For `keywords`, it used to have both `'key1, key2'` of string type, and `['key1', 'key2']` array of string type. They are now all updated to array of strings. That's the reason of numerous changes you see. They are trivial changes applied to the front matter.

File: v3/docs/00-style-and-contributor-guidelines/a-contributor-guidelines/index.mdx [hunk 1]
```diff
@@ -4,7 +4,7 @@ slug: /v3/contribute/style-guide
 version: '3.0'
 section: docs
 category: style guide
-keywords: contribute, style guide
+keywords: ['contribute', 'style guide']
```
Comment (NukeManDan) [inline v3/docs/00-style-and-contributor-guidelines/a-contributor-guidelines/index.mdx:7]: All of these need to be yml syntax.
```suggestion
keywords:
  - contribute
  - style guide
```

**Review by NukeManDan — CHANGES_REQUESTED** [review]

File: v3/docs/00-style-and-contributor-guidelines/a-contributor-guidelines/index.mdx (hunk 1 above)
Comment (NukeManDan) [reply v3/docs/00-style-and-contributor-guidelines/a-contributor-guidelines/index.mdx:7]: example: https://assemble.io/docs/YAML-front-matter.html
https://yaml.org/

Comment (rmnprkrl) [reply v3/docs/00-style-and-contributor-guidelines/a-contributor-guidelines/index.mdx:7]: Both is accepted, differece here is only that the array syntax is an one-liner.. the yaml format makes easier to write more complex data than just scalars but we don't need there. So really depends just on what is your writing preference in md files

**Comment by jimmychu0807** [comment]
@NukeManDan @rmnprkrl frontmatter `keywords` style updated.

**Review by rmnprkrl — APPROVED** [review]
LGTM, should be merger asap (missing img on homepage) @imadarai @sacha-l

**Review by imadarai — APPROVED** [review]

**Pull request merged**